bootstrap/092_service_account_setup.sql

```diff
@@ -18,19 +15,26 @@ begin
         CALL ADMIN.UPGRADE_CHECK();
     grant MONITOR, OPERATE on TASK TASKS.UPGRADE_CHECK to APPLICATION ROLE ADMIN;
 
-    call internal.set_config('tenant_url', :web_url);
-    call internal.set_config('url', :url);
-
-    let tenant_id text := (select split_part(:web_url, '/', -1));
-    call internal.set_config('tenant_id', :tenant_id);
+    -- Merge all three properties into the config table in one merge statement.
+    MERGE INTO internal.config AS target
```

Comment:
I had  a slightly different syntax (some extra brackets I think) did you test this?

Reply:
Yeah, I called the procs locally and it's working. What'd you have in mind?

Looking at this again, I should also take the opportunity to explicitly pass the tenant ID rather than parsing it out.

Reply:
> Yeah, I called the procs locally and it's working. What'd you have in mind?

all good then. Prob a non-event. Just wanted to check that there wasn't something funny going on